luarules/gadgets/dbg_test_framework_proxy.lua

```diff
@@ -2,7 +2,6 @@ function gadget:GetInfo()
 	return {
 		name = "Test Framework Proxy",
 		desc = "Proxy for Synced commands",
-		author = "citrine",
```

Comment:
?

Reply:
I don't like author credits in built-in widgets/gadgets. Just personal preference; I'm fine with `git blame` as credit for myself.